wallet/src/de/schildbach/wallet/ui/main/WalletActivity.java

```diff
@@ -1,25 +1,25 @@
 /*
- * Copyright 2020 Dash Core Group
+ * Copyright 2020 Dash Core Group.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
```

Comment:
I wonder how the license changed from GPL3 to Apache 2.0.  In 2013, before any of us were working on this project, this file was using GPL3.

Good catch.

https://github.com/dashevo/dash-wallet/blob/780fbad36d9d321d3103c973bc708497dac8accd/wallet/src/de/schildbach/wallet/ui/WalletActivity.java

Reply:
That was actually an automatic change Android Studio made when I moved the file :)